src/config/config.h

```diff
@@ -144,6 +144,7 @@ struct Config {
   int pipeline_size;
   int sequence_gap;
 
+  bool redis_cursor_compatible = true;
```

Comment:
I just wondering why the initial value is `true`? As I see in the `.cc` file we have the default value as `false` for this variable.
Other LGTM.

Reply:
Fixed